### PR TITLE
Added media library demo to media-library route.

### PR DESCRIPTION
**Pull request by saffabook**

### CODE REVIEW

File: app/scripts/routes/media-library/media-library-list/media-library-list.html [hunk 1]
```diff
@@ -1 +1,18 @@
-Start here
+<div  ng-controller="mediaLibraryCtrl as media" ng-cloak>
+    <md-subheader class="md-no-sticky">
+        <a href="" class="breadcrumbs" ng-repeat="folder in breadcrumbsFolders" ng-click="openBreadcrumbsFolder($index)">{{folder.name}}/</a>
```
Comment (saffabook) [inline app/scripts/routes/media-library/media-library-list/media-library-list.html]: This class will become a SUIT css component. So remove for now.

File: app/scripts/routes/media-library/media-library-list/media-library-list.html [hunk 2]
```diff
@@ -1 +1,18 @@
...
+    </md-subheader>
+    <md-list  ng-repeat="media in files.contain track by $index">
+        <md-list-item class="noright" ng-if="media.type == 'folder'" ng-click="openFolder(media)">
+            <img alt="{{ media.name }}" src="images/folder-512.png" class="md-avatar">
```
Comment (saffabook) [inline app/scripts/routes/media-library/media-library-list/media-library-list.html]: We are going to add icons for this. We can discuss this. So lets delete the image for this.

File: app/scripts/routes/media-library/media-library-list/media-library-list.html [hunk 3]
```diff
@@ -1 +1,18 @@
-Start here
+<div  ng-controller="mediaLibraryCtrl as media" ng-cloak>
+    <md-subheader class="md-no-sticky">
+        <a href="" class="breadcrumbs" ng-repeat="folder in breadcrumbsFolders" ng-click="openBreadcrumbsFolder($index)">{{folder.name}}/</a>
+    </md-subheader>
+    <md-list  ng-repeat="media in files.contain track by $index">
+        <md-list-item class="noright" ng-if="media.type == 'folder'" ng-click="openFolder(media)">
```
Comment (Rillus) [inline app/scripts/routes/media-library/media-library-list/media-library-list.html]: What does the class `noright` do?

Might be good to include a screengrab with PRs so we can see what you've built?

**Comment by saffabook** [comment]
Can we merge this yet?

**Comment by HelenDobosh** [comment]
http://joxi.net/DmBg3ZncN1VGM2

File: app/scripts/routes/media-library/media-library-list/media-library-list.html (hunk 1 above)
Comment (HelenDobosh) [reply app/scripts/routes/media-library/media-library-list/media-library-list.html]: removed class

File: app/scripts/routes/media-library/media-library-list/media-library-list.html (hunk 2 above)
Comment (HelenDobosh) [reply app/scripts/routes/media-library/media-library-list/media-library-list.html]: changed image to icon

File: app/scripts/routes/media-library/media-library-list/media-library-list.html (hunk 3 above)
Comment (HelenDobosh) [reply app/scripts/routes/media-library/media-library-list/media-library-list.html]: removed class because we don't need it

**Comment by saffabook** [comment]
Looks good to me, merge it :)

**Pull request merged**